pkg/controller/kibana/config/settings.go

```diff
@@ -15,10 +15,20 @@ import (
 	"github.com/elastic/cloud-on-k8s/pkg/controller/common/settings"
 	"github.com/elastic/cloud-on-k8s/pkg/controller/kibana/es"
 	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/rand"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 // Kibana configuration settings file
-const SettingsFilename = "kibana.yml"
+const (
+	SettingsFilename = "kibana.yml"
+	name             = "kibana-config"
```

Comment:
took me some time this is used for logging 😄 
I think I would either hardcode it in `logf.Log.WithName(name)` or try to find a more explicit name for the const?

Reply:
Ah I see in other places we just hardcode it as you suggested, I think the first example I looked at had this pattern so mimicked that. It looks like it is about 50/50 which we do.